requirements.txt

```diff
@@ -2,6 +2,14 @@ amazon.ion~=0.7.0
 boto3~=1.16.56
```

Comment:
I think it's worth labeling these top two sections with `# Direct dependencies` and `# Testing dependencies`.

Reply:
Added